tidb-cloud/tidb-cloud-billing.md

```diff
@@ -7,9 +7,93 @@ summary: Learn about TiDB Cloud billing.
 
 > **Note:**
 >
-> [TiDB Cloud Starter clusters](/tidb-cloud/select-cluster-tier.md#tidb-cloud-starter) are free until April 30, 2025, with a 100% discount off. After that, usage beyond the [free quota](/tidb-cloud/select-cluster-tier.md#usage-quota) will be charged.
+> [TiDB Cloud Starter clusters](/tidb-cloud/select-cluster-tier.md#tidb-cloud-starter) are free until June 9, 2025, with a 100% discount off. After that, usage beyond the [free quota](/tidb-cloud/select-cluster-tier.md#usage-quota) will be charged.
 
-TiDB Cloud charges according to the resources that you consume. You can visit the [TiDB Cloud Starter Pricing Details](https://www.pingcap.com/tidb-serverless-pricing-details/) pages to get more information about the pricing.
+TiDB Cloud charges according to the resources that you consume.
+
+## Pricing
+
+TiDB Cloud offers the following two pricing options to suit different needs:
+
+* TiDB Cloud Starter: you are charged based on the number of Request Units (RUs) consumed by your application.
+* TiDB Cloud Essential: you are charged based on the number of provisioned Request Capacity Units (RCUs), **not** on the actual usage by your application.
+
+### Request Unit (RU) and Request Capacity Unit (RCU)
+
+A **Request Unit (RU)** is a unit of measure used to represent the amount of resources consumed by a single request to the database. The amount of RUs consumed by a request depends on various factors, such as the operation type or the amount of data being retrieved or modified.
+
+A **Request Capacity Unit (RCU)** is a unit of measure used to represent the provisioned compute capacity for your TiDB Cloud Essential cluster. One RCU provides a fixed amount of compute resources that can process a certain number of RUs per second. The number of RCUs you provision determines your cluster's baseline performance and throughput capacity.
+
+Currently, the RU includes statistics for the following resources:
+
+| Resource        | RU Consumption                                 |
+|-----------------|-------------------------------------------------|
+| Read            | 2 storage read batches consume 1 RU             |
+|                 | 8 storage read requests consume 1 RU            |
+|                 | 64 KiB read request payload consumes 1 RU       |
+| Write*          | 2 storage write batches consume 1 RU             |
+|                 | 2 storage write requests consume 1 RU           |
+|                 | 2 KiB write request payload consumes 1 RU       |
+|                 | 16 KiB write request payload consumes 1 RU (for transactions** >= 16MiB) |
+| SQL CPU         | 3 ms consumes 1 RU                              |
+| Network Egress  | 1 KiB read consumes 1 RU for public endpoint    |
+|                 | 4 KiB read consumes 1 RU for private endpoint   |
```

Comment:
I'll convert this Markdown table into HTML format, to avoid possible confusion. Then I'll merge this PR.